src/app/tasks/checker-inbox-and-tasks-tabs/loan-approval-tabs/loans-pending-kyc-verification/loans-pending-kyc-verification.component.ts

```
@@ -0,0 +1,218 @@
+        kycRejectionComment,
+        markClientFailedKYCVerification
+      };
+      const selectedAccounts = this.selection.selected.length;
+      const listSelectedAccounts = this.selection.selected;
+      let approvedAccounts = 0;
+      this.batchRequests = [];
+      listSelectedAccounts.forEach((element: any) => {
+      const bodyData = JSON.stringify(formData);
+      this.tasksService.executeMakerCheckerRejectLoanAction(element.id, formData).subscribe((response: any) => {
+        response.forEach((responseEle: any) => {
+          if (responseEle.statusCode = '200') {
```

Biome (2.1.2)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)